lib/graphql/define/assign_argument.rb

```diff
@@ -10,8 +10,11 @@ def self.call(target, name, type, description = nil, default_value: nil, &block)
         end
         argument.name = name.to_s
         argument.type = type
-        argument.description = description
-        argument.default_value = default_value
+
+        description && argument.description = description
+        # check nil because false is a valid value
+        default_value != nil && argument.default_value = default_value
```

Comment:
Sure, that's equivalent. Do you want me to amend my commit, since this change is small?